### DISCUSSION TITLE
Disable automatic execution of update scripts

### DISCUSSION THREAD

**Discussion by Flyingmana** [OpenMage/magento-lts#2155] [Ideas]
instead to automatic execute update scripts when the config cache is rebuild, they should be triggered in a controlled workflow.

- [ ] disable automatic update script execution by default
- [ ] add notice in Admin Area, if update scripts are pending
- [ ] Add Button/Action to trigger updates via Admin area
- [ ] add cli script to trigger Update scripts
- [ ] add documentation

**Comment by tmotyl** [comment]
FYI, There is an xml config disabling update scripts already built in

**Comment by Flyingmana** [comment]
Iam aware, and for 2 of the sub items I think @dng-dev has a ready solution.

But i think it still makes sense to combine this with something in the Admin are(multiple PullRequests would be ok for this)

**Comment by midlan** [comment]
I think this feature should by done by custom module, not updating magento core, since lot of people have their build pipelines relying on the automatic install of updates.

**Comment by colinmollenhour** [comment]
Automatic install is a really bad thing to enable on production due to race conditions and possibility that a minor upgrade failure could cause a major snafu. What about once some improved tooling is included out of the box placing a notice in the admin panel advising the user to disable automatic updates?
